packages/grafana-ui/src/components/Menu/MenuGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useStyles } from '../../themes';
+import { MenuItemProps } from './MenuItem';
+
+/** @internal */
+export interface MenuItemsGroup {
+  /** Label for the menu items group */
+  label?: string;
+  /** Items of the group */
+  items: MenuItemProps[];
+}
+/** @internal */
+export interface MenuGroupProps extends Partial<MenuItemsGroup> {
+  /** special children prop to pass children elements */
+  children?: React.ReactNode;
```

Comment:
Again- think this should be a required property, why would you render a menbu group without items?